build-tools/packages/build-cli/package.json

```diff
@@ -31,7 +31,7 @@
 		"build:full:compile": "npm run build:full",
 		"build:genver": "gen-version",
 		"build:machine-diagram": "jssm-viz -i \"./src/machines/*.fsl\"",
-		"build:manifest": "oclif manifest",
+		"build:manifest": "cross-env NODE_OPTIONS='--experimental-abortcontroller' oclif manifest",
```

Comment:
why are node options required here?

Reply:
It's a workaround for https://github.com/oclif/core/issues/630. I'll add notes to the readme.